index.html

```diff
@@ -1,8 +1,73 @@
 <!DOCTYPE html>
 <html>
    <head>
+      <title>Now Rosemarie Is Six</title>
+      <link rel="stylesheet" type="text/css" href="styles.css"/>
    </head>
    <body>
-      <!-- Write your HTML in this file! -->
+      <header>
+         <article>
+            <h1 id="mainHeading">Now We Are Six</h1>
+           <div class="extra-padding">
+            <p id= "testP" class="funParagraph" >My beautiful Rosemarie was born in April 2017. 
+               Three days of labor and she came out on her own time. 
+               She just turned five and I started thinking about her sixth birthday and this poem.
+               My aunt gave me A.A. Milne's book when I turned six, and my partner and I will do the same for Rosemarie.
+               She continues to grow and develop her own personality, as we all do beyond the age of six. 
+               I love her empathy, her curiosity, her bravery, her independence. 
+               She is my love bug, and I dedicate his page to her.</p>
+            <p id="testP" class="align-right funParagraph">To my Rosemarie <br>- Love, your Mommy.
+            </p>
+         </div>
+         </article>
+      </header>
+
```

Comment:
I like the spacing after this header tag, it makes the code a little more readable vs if it was all squished together.